src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,131 @@
+syntax = "proto3";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor, normalized to [0,1]
+    float break_beam_reading = 2;
```

Comment:
Similar to below, why are we sending both the 'ball_in_break_beam' and 'break_beam_reading'? Could you not get the 'ball in break beam' data from the reading?

Reply:
It might be good for debugging to report break_beam_reading, but for actual use, we only need ball_in_break_beam